README.md

```diff
@@ -553,7 +555,28 @@ var elements = minim.fromCompactRefract(['category', {}, {}, [
 console.log(elements.get(0).content); // hello, world
 
 // Unregister your custom element
-minim.registry.unregister('category');
+minim.unregister('category');
+```
+
+#### Creating Namespace Plugins
+
+It is also possible to create plugin modules that define elements for custom namespaces. Plugin modules should export a single `namespace` function that takes an `options` object which contains an existing namespace to which you can add your elements:
+
+```javascript
+var minim = require('minim').namespace();
+
+// Define your plugin module (normally done in a separate file)
+var plugin = {
+  namespace: function(options) {
+    var base = options.base;
+    var ArrayElement = base.getElementClass('array');
+
+    base.register('category', ArrayElement);
```

Comment:
One thought here is that there is now a lot of boilerplate that has to happen in plugins. You have to check to make sure `base` exists, and `getElementClass` in order to even register something. Would be nice to have a helper that did some of this for you.

```js
register('category', 'array', options);
```

Of course, not necessary for this PR, just thinking out loud.